### PR TITLE
feat(ci/wercker): Add wercker ci to halyard

### PR DESCRIPTION
**Pull request by guoyongzhang**
Added wercker as a ci provider.

### CODE REVIEW

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/wercker/WerckerAddMasterCommand.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2017, 2018, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/wercker/WerckerAddMasterCommand.java]: Is 2017 correct?

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/wercker/WerckerEditMasterCommand.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      names = "--token",
+      password = true,
+      description = WerckerCommandProperties.TOKEN_DESCRIPTION
+  )
+  public String token;
+
+  @Parameter(
+	      names = "--organizations",
+	      variableArity = true,
+	      description = WerckerCommandProperties.ORGANIZATIONS_DESCRIPTION
```
Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/wercker/WerckerEditMasterCommand.java]: Indentation here doesn't match the rest of the file

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/wercker/WerckerAddMasterCommand.java (hunk 1 above)
Comment (guoyongzhang) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/wercker/WerckerAddMasterCommand.java]: According to Oracle policy, yes.

**Comment by guoyongzhang** [comment]
There will be more PRs for wercker support in deck, igor, orca, etc.  But, this PR doesn't depend on them.

**Comment by desagar** [comment]
PRs have been put up for the Wercker CI and Wercker pipeline stage feature implementation. Related PRs: 

https://github.com/spinnaker/igor/pull/297
https://github.com/spinnaker/deck/pull/5519
https://github.com/spinnaker/echo/pull/295
https://github.com/spinnaker/gate/pull/574
https://github.com/spinnaker/orca/pull/2313

**Comment by sbwsg** [comment]
Support's been merged in to Spinnaker now so I'm proceeding with merge to Halyard.

**Pull request merged**